pulse/service/src/main/java/gov/ca/emsa/pulse/service/PatientService.java

```diff
@@ -45,10 +47,14 @@
 	@RequestMapping(value = "/search", method = RequestMethod.POST)
 	public @ResponseBody Query searchPatients(@RequestBody PatientSearch patientSearchTerms) throws JsonProcessingException {
 		Query returnQuery;
+		Pattern p = Pattern.compile("\\d{4}([01]\\d([0-3]\\d([0-2]\\d([0-5]\\d([0-5]\\d)?)?)?)?)?([+-]\\d{4})?$");
```

Comment:
Do you have any tests that run against this code? If there are, I'd add a bunch of good/bad date/time strings to those tests so future developers could see what format we're looking for here